addons/payment/models/payment_transaction.py

```diff
@@ -602,6 +602,7 @@ def _create_refund_transaction(self, amount_to_refund=None, **custom_create_valu
             'operation': 'refund',
             'source_transaction_id': self.id,
             'partner_id': self.partner_id.id,
+            'token_id': self.token_id.id,
             **custom_create_values,
```

Comment:
To follow the fields' order.
```suggestion
            'token_id': self.token_id.id,
            'operation': 'refund',
            'source_transaction_id': self.id,
            'partner_id': self.partner_id.id,
            **custom_create_values,
```

Reply:
@AntoineVDV  how do I find the version the refund feature was merged in ?

Reply:
Find the commit responsible for the feature and, from there, you can access the PR that merged the commit. The PR will have a tag 15.1 or 15.2.